### ISSUE TITLE
Exit shortly after starting

### ISSUE THREAD

**Issue by shouldsee** [eneshecan/whatsapp-for-linux#289]
Thanks for sharing and maintaining this great software!!!

**Description**

Exit shortly after starting with the log.

```
<13>May  6 00:22:56 whatsapp-for-linux[9938]: SettingMap: Key file does not have group “web”, returning default value: 1
<13>May  6 00:22:56 whatsapp-for-linux[9938]: SettingMap: Key file does not have group “web”, returning default value: 0
<13>May  6 00:22:56 whatsapp-for-linux[9938]: SettingMap: Key file does not have group “general”, returning default value: 1
<13>May  6 00:22:56 whatsapp-for-linux[9938]: SettingMap: Key file does not have group “web”, returning default value: 0
<13>May  6 00:22:56 whatsapp-for-linux[9938]: SettingMap: Key file does not have group “general”, returning default value: 0
<13>May  6 00:22:56 whatsapp-for-linux[9938]: SettingMap: Key file does not have group “general”, returning default value: 1
<13>May  6 00:22:56 whatsapp-for-linux[9938]: SettingMap: Key file does not have group “general”, returning default value: 0
<13>May  6 00:22:56 whatsapp-for-linux[9938]: SettingMap: Key file does not have group “general”, returning default value: 0
<13>May  6 00:22:56 whatsapp-for-linux[9938]: SettingMap: Key file does not have group “appearance”, returning default value: 0
<13>May  6 00:23:01 whatsapp-for-linux[9938]: whatsapp-for-linux: symbol lookup error: whatsapp-for-linux: undefined symbol: webkit_web_view_get_is_web_process_responsive

```

**To Reproduce**
Start program, and wait for it to load
1. Steps to reproduce the behavior:
1. See the error

**Expected behavior**
no exit

**Screenshots**
see log

**Environment**
 - Distro [e.g. Ubuntu, Arch Linux]
```
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="Ubuntu 18.04.1 LTS"
```

 - Package .deb `apt install ./whatsapp-for-linux_1.6.2_amd64.deb`
 - Desktop Environment or Window Manager `lightdm` ?

**Comment by eneshecan** [comment]
Hi @shouldsee. Thanks for reporting! I think this is because Ubuntu 18.04 has a very old webkitgtk version.

**Comment by shouldsee** [comment]
Hi thank you @eneshecan ! Indeed `snap install whatsapp-for-linux` after apt remove solved the problem! I can now using 1.6.2!
